Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Control Flow Graph: Fix modifiers not being analyzed
```

Comment:
This covers way much more than the bug.

Reply:
What about something like `Perform proper virtual lookup for modifiers for uninitialized variable and unreachable code analysis.` - does it currently do more than these two?

Reply:
Nope that about covers it. The rest is just refactoring